src/main/java/uk/gov/hmcts/reform/blobrouter/clients/BulkScanProcessorClient.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.blobrouter.clients;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@FeignClient(
+    name = "bulk-scan-processor-client",
+    url = "${bulk-scan-processor-url}"
+)
+public interface BulkScanProcessorClient {
+
+    @GetMapping(value = "/token/{container}", consumes = APPLICATION_JSON_VALUE)
```

Comment:
Tests say the url part is `service`. Here it's `container`.
Not sure which one is correct, but should definitely be consistent.

Reply:
in processor it is service. changed toservice